beetsplug/discogs.py

```diff
@@ -200,7 +200,7 @@ def get_albums(self, query):
         query = re.sub(r'(?u)\W+', ' ', query).encode('ascii', "replace")
         # Strip medium information from query, Things like "CD1" and "disk 1"
         # can also negate an otherwise positive result.
-        query = re.sub(br'(?i)\b(CD|disc)\s*\d+', '', query)
+        query = re.sub(b'(?i)\b(CD|disc)\s*\d+', '', query)
```

Comment:
I'm confused by this. as we want the `r'`, and `br'` should mean bytes raw.  At least according to this https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals  



> New in version 3.3: The 'rb' prefix of raw bytes literals has been added as a synonym of 'br'.